templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<p>Which notebook is final: this one or the v2 notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
This one (named `ez_segmenter.ipynb`). The v2 file can be removed.